LibVLCSharp/LibVLCSharp.csproj

```diff
@@ -62,9 +63,14 @@
     <Compile Include="Platforms\tvOS\**\*.cs" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net40' ">
-    <PackageReference Include="System.ValueTuple" Version="4.5.0"/>
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
     <PackageReference Include="Microsoft.Bcl.Async" Version="1.0.168" />
   </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework.StartsWith('uap'))">
+    <Compile Include="Platforms\UWP\**\*.cs" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="SharpDX.Direct3D11" Version="4.2.0" />
```

Comment:
For further reference, SharpDX has just been announced as not maintained anymore. We chose to use it anyway, given the maturity of the project. Maybe that decision will be reconsidered later.

Reply:
Yes.